src/classes/CW_MessagingApprovalTrigHandlerTest.cls

```diff
@@ -0,0 +1,82 @@
+/**
+* File Name   :    CW_MessagingApprovalTrigHandlerTest
+* Description :    trigger to update asset date approval process whenever related messaging records got changed
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 20/08/2014Suneetha Jasthi     B-122329  Mailer/teaser/SMS messaging approval processes
+*
+*/
+
+
+@isTest
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.